configure

```diff
@@ -1989,6 +1993,25 @@ Available commands are:
       message.index("\n") != nil
     end
   end
+
+  # Returns true if MacPorts is installed in its default location.
+  def macports?
+    File.exists? '/opt/local/bin/port'
```

Comment:
Also, while the argument of duplicate `port` commands in PATH makes sense, this wouldn't work anyway due to the shell commands using a non-absolute path for this command.